Assets/Scripts/Game/FPSWeapon.cs

```diff
@@ -540,6 +545,14 @@ private void LoadWeaponAtlas()
                 {
                     textures.Add(GetWeaponTexture2D(filename, record, frame, metalType, out rect, border, dilate));
 
+                    if (moddedWeaponHUDAnimsEnabled && SpecificWeapon != null)
+                    {
+                        filename = WeaponBasics.GetModdedWeaponFilename(SpecificWeapon);
+
+                        if (filename == "")
```

Comment:
Please use string.IsNullOrEmpty for these checks.

Reply:
Changed the if-statement check to instead use, string.IsNullOrEmpty(filename)